cmd/crossplane/core/core.go

```diff
@@ -103,8 +103,9 @@ type startCommand struct {
 	MaxReconcileRate                 int           `default:"100" help:"The global maximum rate per second at which resources may checked for drift from the desired state."`
 	MaxConcurrentPackageEstablishers int           `default:"10"  help:"The the maximum number of goroutines to use for establishing Providers, Configurations and Functions."`
 
-	WebhookEnabled                      bool `default:"true"  env:"WEBHOOK_ENABLED"                        help:"Enable webhook configuration."`
-	AutomaticDependencyDowngradeEnabled bool `default:"false" env:"AUTOMATIC_DEPENDENCY_DOWNGRADE_ENABLED" help:"Enable automatic dependency version downgrades. This configuration requires the 'EnableDependencyVersionUpgrades' feature flag to be enabled."`
+	WebhookEnabled                    bool `default:"true"  env:"WEBHOOK_ENABLED"                      hidden:""`
+	EnableWebhook                     bool `default:"true"  env:"ENABLE_WEBHOOK"                       help:"Enable webhook configuration."`
```

Comment:
Nit: Please name this `EnableWebhooks` and `ENABLE_WEBHOOKS`, since these days we have more than one.